### PR TITLE
Block Out user interface; fixes #2

### LINKED ISSUE #2: Block out user interface
The user interface needs to be blocked out for adapter operation.
Create a UI without business logic.

### PR DESCRIPTION
**Pull request by nathanatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/DEHP-ECOSIMPRO/pulls) open
- [x] I have verified that I am following the DEHP-EcosimPro [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/DEHP-ECOSIMPRO/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description

; fixes #2
- [x] added Views
- [x] added DEHPCommon package
- [x] Autofac

image

### CODE REVIEW

File: DEHPEcosimPro/Enumerators/MappingDirection.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+namespace DEHPEcosimPro.Enumerators
+{
+    /// <summary>
+    /// Enumeration of the two direction a mapping can be done
+    /// </summary>
+    public enum MappingDirection
+    {
+        /// <summary>
+        /// Left means from EcosimPro to 10-25 source
```
Comment (samatrhea) [inline DEHPEcosimPro/Enumerators/MappingDirection.cs:33]: the names are not clear and need docs to be explanatory, i find the left-to-right idea a bit odd. Can you make the directionality more explicit please? And, is this not something that is more for Common? All DST adaptors will have a MappingDirection where on the one side we have 10-25 and the other we have a DST.

File: DEHPEcosimPro/Views/Rows/MappingThing.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="MappingRow.xaml.cs" company="RHEA System S.A.">
```
Comment (samatrhea) [inline DEHPEcosimPro/Views/Rows/MappingThing.xaml.cs:2]: this does not correspond to the class name, also, can i suggest to make it `MappingThingControl` ?

File: appveyor.yml [hunk 3]
```diff
@@ -11,6 +11,10 @@ environment:
        SONARCLOUD_URL: 'https://sonarcloud.io'
        SONARCLOUD_TOKEN:
          secure: th/5nKHJ0jW1sM3a0bs0cohRLIeMAx4Y5Do/al3L11XbWHOgoCdNYrNBQmSo9tFs
+       # To be changed in the future since this token belongs to a personal account. See below.
+       # https://github.com/RHEAGROUP/DEHP-Common/wiki/Build-And-Release#modifying-the-github-token
+       GITHUB_TOKEN:
+         secure: uS7/WlBplrAs/j9ab9P3SUN7GZyf4QAHZX//YpDOzXG/DZ6o1MGHdIWLHHdDYM5P
```
Comment (samatrhea) [inline appveyor.yml:17]: this should indeed **not** be here... please add a ticket somewhere so this is captured that work needs to be done ASAP

**Review by samatrhea — CHANGES_REQUESTED** [review]

**Review by lxatrhea — APPROVED** [review]
Only one question: Should all the ThingIcon stuff be moved to DEHPCommon?

**Review by samatrhea — APPROVED** [review]

File: DEHPEcosimPro/Views/DataSourcePanel.xaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+﻿<UserControl x:Class="DEHPEcosimPro.Views.DataSourcePanel"
```
Comment (alexatrhea) [inline DEHPEcosimPro/Views/DataSourcePanel.xaml:1]: not clear what this does, perhaps rename?

File: DEHPEcosimPro/Views/Rows/MappingThingControl.xaml [hunk 5]
```diff
@@ -0,0 +1,73 @@
+﻿<UserControl x:Class="DEHPEcosimPro.Views.Rows.MappingThingControl"
```
Comment (alexatrhea) [inline DEHPEcosimPro/Views/Rows/MappingThingControl.xaml:1]: common perhaps as well?

File: DEHPEcosimPro/Views/SplashScreen.xaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
+﻿<dx:SplashScreenWindow x:Class="DEHPEcosimPro.Views.SplashScreen"
```
Comment (alexatrhea) [inline DEHPEcosimPro/Views/SplashScreen.xaml:1]: Common?

File: DEHPEcosimPro/Views/StatusBarControl.xaml [hunk 7]
```diff
@@ -0,0 +1,32 @@
+﻿<StatusBar x:Class="DEHPEcosimPro.Views.StatusBarControl"
```
Comment (alexatrhea) [inline DEHPEcosimPro/Views/StatusBarControl.xaml:1]: common?

**Review by alexatrhea — CHANGES_REQUESTED** [review]
Quite a bit of refactoring needed. Please also in general place some sort of visual indicator in the main window that shows the direction of transfer.

**Review by alexatrhea — APPROVED** [review]

**Pull request merged**